seed/requirements.yaml

```diff
@@ -1,7 +1,5 @@
-# FIXME https://github.com/edgelevel/gitops-k8s/issues/6
 dependencies:
-  - name: argo-cd
-    alias: argocd
-    version: 0.2.2
-    repository: https://argoproj.github.io/argo-helm
+  - name: argocd
+    version: 1.0.0-0
```

Comment:
`-0` ???

Reply:
Please have a look at [this](https://github.com/edgelevel/helm-charts/pull/2#discussion_r287355342)

Reply:
@niqdev I had a quick look into versioning this morning. Looks like they have an idiom for this already.
`version: A SemVer 2 version (required)` <- this is the chart version
`appVersion: The version of the app that this contains (optional). This needn't be SemVer.` This is the application version i.e. version 1.0,0 of ArgoCD

https://github.com/helm/helm/blob/master/docs/charts.md#the-chartyaml-file

Reply:
Interesting [this](https://github.com/helm/helm/blob/master/docs/charts.md#the-appversion-field), but
```
This field is informational, and has no impact on chart version calculations.
```